server/branch.ts

```diff
@@ -213,7 +231,7 @@ export class BranchConfig {
 			await this.loadAllBranches();
 			return true;
 		}
-		catch {
+		catch (err) {
```

Comment:
TypeScript 2.5.x supports this type of try / catch syntax if you don't need the error variable